pkgs/development/libraries/science/chemistry/cppe/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "cppe";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "maxscheurer";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-guM7+ZWDJLcAUJtPkKLvC4LYSA2eBvER7cgwPZ7FxHw=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  cmakeFlags = [ "-DCMAKE_INSTALL_LIBDIR=lib" ];
+
+  meta = with lib; {
+    description = "C++ and Python library for Polarizable Embedding";
+    homepage = "https://github.com/maxscheurer/cppe";
+    license = licenses.lgpl3Only;
+    platforms = platforms.linux;
```

Comment:
Not tested and there seems to be no clear rule how to deal with this? I've changed to `[ "x86_64-linux" "x86_64-darwin" ]` for all, but it would likely also run with `aarch64` stuff (at least I don't see any reason why it shouldn't. What do you think?

Reply:
Will resolve the others on favour of this and change them according to what we figure out here.

Reply:
Alrighty, changed it to `platforms.unix`, as there is no reason it would be tied to specific architecture.